src/sdk/createClientWithFundedWallet.js

```diff
@@ -18,44 +13,37 @@ const wait = require('../util/wait');
  */
 async function createClientWithFundedWallet(network, faucetPrivateKeyString, seed = undefined) {
   // Prepare to fund wallet
-  const faucetPrivateKey = PrivateKey.fromString(faucetPrivateKeyString);
-  const faucetAddress = faucetPrivateKey
-    .toAddress(network)
-    .toString();
+  const faucetPrivateKey = faucetPrivateKeyString;
 
-  const options = {
-    wallet: {
-      mnemonic: null,
-    },
-    network,
+  const clientOpts = {
+    network: process.env.NETWORK,
   };
 
   if (seed) {
-    options.seeds = [seed];
+    clientOpts.seeds = [seed];
   }
 
-  const dashClient = new Dash.Client(options);
-
-  const account = await dashClient.getWalletAccount();
+  const faucetClient = new Dash.Client({
```

Comment:
Let's just instantiate just a Wallet?

Reply:
I'm actually not sure it would really be much of a overhead actually, we removed all init from dash-sdk, but I think it's just a better usage too. 
Dash.Wallet. 

On it !